contributors/devel/sig-release/cherry-picks.md

```diff
@@ -42,6 +42,11 @@ Again, the emphasis is on critical bug fixes, e.g.,
 - Panic, crash, hang
 - Security
 
+As a counter example, we would not support backporting a bug fix to an alpha
+feature in general. One condition where we would consider is if that bug caused
+an issue for in another part of the system even when the alpha feature was
+switched off / not in use.
+
```

Comment:
sorry for the post-merge comments, a few nits...

* consider avoiding "we" language
* A legitimately critical issue like data loss or a security issue could still potentially be a reason for a backport for something even in the alpha state, depending on risk to the rest of the system

Would language like this map more closely to our current backport standards?

> A bugfix for a functional issue (not a data loss or security issue) that only affects an alpha feature does not qualify as a critical bug fix.

Reply:
@liggitt please go ahead and open another PR. +1 from me

Reply:
+1